image/tarexport/load.go

```diff
@@ -77,6 +78,9 @@ func (l *tarexporter) Load(inTar io.ReadCloser, outStream io.Writer, quiet bool)
 		if err != nil {
 			return err
 		}
+		if img.OS != runtime.GOOS {
+			return fmt.Errorf("cannot load %s image on %s", img.OS, runtime.GOOS)
```

Comment:
Looks like the check should be less strict, and also accept `img.OS == ""`, or just change this to a Warning.

Current CI is failing because of this, because the `empty` image apparently doesn't have an `OS`;

```
02:30:05 ---> Making bundle: .ensure-emptyfs (in bundles/17.06.0-dev/test-integration-cli)
02:30:05 ++++ tar -cC bundles/17.06.0-dev/test-integration-cli/emptyfs .
02:30:05 ++++ docker load
02:30:05 cannot load  image on linux
```

Could there be other cases where `OS` doesn't have to match actual OS? (idk, things could be emulated inside the container or VM)